src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -93,32 +79,41 @@ public static void addDocumentToBundle(FooterService footerService, SscsCaseData
     }
 
     public static DocumentType getPostHearingReviewDocumentType(PostHearing postHearing, boolean isPostHearingsEnabled) {
+        if (isPostHearingsEnabled && nonNull(postHearing.getReviewType())) {
+            return getPostHearingReviewDocumentType(postHearing);
+        }
+
+        return DocumentType.DECISION_NOTICE;
+    }
+
+    private static DocumentType getPostHearingReviewDocumentType(PostHearing postHearing) {
         PostHearingReviewType postHearingReviewType = postHearing.getReviewType();
-        if (isPostHearingsEnabled && nonNull(postHearingReviewType)) {
```

Comment:
Why have you removed the post hearing flag?

Reply:
i havent, its just the diff, its stayed on line 82